### PR TITLE
Awesome books: style core elements

### PR DESCRIPTION
**Pull request by abrormukimov**
# In this project, I:
- add styling to the UI;

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,8 +10,9 @@
 <body>
```
Comment (rloterh) [inline index.html:10]: - [ ] The `package.json` and `package-lock.json` files are missing in your project directory. Kindly ensure to include these files in the project uploaded to the repository so as to enable cloning of your project by repo visitors

File: app.js [hunk 2]
```diff
@@ -23,14 +22,13 @@ class Book {
     remove.setAttribute('data-title', `${book.title}`);
 
     title.textContent = `${book.title}`;
-    author.textContent = `${book.author}`;
```
Comment (rloterh) [inline app.js:26]: - [ ] OPTIONAL
The book _title_ and book _author_ input are both accepting **numbers** as inputs. Kindly endeavor to provide validation for the same.

**Review by rloterh — APPROVED** [review]
Hi Abror & Lilian, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (abrormukimov) [reply index.html:10]: Why are you asking to do something which is not in the project requirements. I hope you will have a look at them first

File: app.js (hunk 2 above)
Comment (abrormukimov) [reply app.js:26]: this one as well.

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html:10]: Hello Team, kindly note that the above-mentioned files are basic requirements for your project type and are **not optional**. Without these files, code reviewers find it difficult to find out exactly how your application is functioning. We, therefore, expect you to add these files to your project without explicitly telling you to do so... the reason why you don't see that mentioned in this project's requirement. I therefore kindly entreat you painstakingly add these files to your project. Cheers and Happy coding!

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @abrormukimov  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**